modules/update_folders.nf

```diff
@@ -11,7 +11,7 @@ process UPDATE_FOLDERS {
     path docker_log_file
 
     output:
-    val "ready"
+    path "output_annotation.json"
```

Comment:
This might be a problem in cases where an INVALID predictions file is being passed thru the pipeline. Investigate.

Reply:
Update: This is not a problem because the Docker execution log will always be created and uploaded even if the INVALID file is not, so the `output_annotation.json` will always be created.